content/cookbook/development/connect-environment.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Connecting to a Solana Environment
+sidebarSortOrder: 2
+description: "Learn how to connect to a Solana environment."
+---
+
+When you are working on Solana development, you will need to connect to a
+specific RPC API endpoint. Solana has 3 public development environments:
+
+- mainnet-beta https://api.mainnet-beta.solana.com
+- devnet https://api.devnet.solana.com
+- testnet https://api.testnet.solana.com
+
+```typescript filename="connect-to-environment.ts"
+import { clusterApiUrl, Connection } from "@solana/web3.js";
+
+(async () => {
+  const connection = new Connection(clusterApiUrl("mainnet-beta"), "confirmed");
+})();
```

Comment:
Per the other PRs you can slip the IIFE and use `esrun` for this. Up to you though, you may not want to ask people to install new tools.

Reply:
For now let's leave this as IIFE, after code-import happens we should get back on it.
Earlier there was a preview code to not show the full code.

apparently after code-import this should be refactored in such a way that IIFE is not visible to end users,
and people can copy exact line instead of copying whole script.